build.gradle

```
@@ -54,6 +55,8 @@ dependencies {
     implementation "com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations"
     // used to connect to umb
     implementation "org.springframework.boot:spring-boot-starter-activemq"
+    // used to parse events generated by console.redhat.com (Export Service)
+    implementation "com.redhat.cloud.event:event-schemas:1.4.10"
```

Please put this dependency version in `dependencies.gradle`, so that it can be centrally managed and updated by dependabot.
```suggestion
    implementation libraries["event-schemas"]
```